modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql

```diff
@@ -24,3 +24,12 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
 
 CREATE INDEX IDX_SESSION_ID ON IDN_AUTH_USER_SESSION_MAPPING (SESSION_ID)
 /
+
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
+/
+
+CREATE INDEX IDX_SPI_APP ON SP_INBOUND_AUTH(APP_ID)
+/
+
+CREATE INDEX IDX_IOP_TID_CK ON IDN_OIDC_PROPERTY(TENANT_ID,CONSUMER_KEY)
+/
```

Comment:
Shall we add a new line at the end of the file? Please do the same for others as well.

Reply:
done